lua/classdrop/server/functions.lua

```diff
@@ -16,30 +16,17 @@ if SERVER then
 
         drop:SetNWInt("customClass", ply:GetCustomClass())
         
-        local tbl = {}
+        local oldClassCooldownTS = ply:GetClassCooldownTS()
+        local oldClassCooldown = ply:GetClassCooldown()
+        ply:UpdateClass(nil)
         
-        if ply.classWeapons then
```

Comment:
why removing this?

Reply:
Because classWeapons are removed and added automatically with the Heroes Core (through UpdateClass)

Reply:
Alright, ty :)